particle-iot/particle-cli
# feature/products

## Description

The combined changes from:

https://github.com/particle-iot/particle-cli/pull/570
https://github.com/particle-iot/particle-cli/pull/565
https://github.com/particle-iot/particle-cli/pull/559
https://github.com/particle-iot/particle-cli/pull/558
https://github.com/particle-iot/particle-cli/pull/553
https://github.com/particle-iot/particle-cli/pull/551
https://github.com/particle-iot/particle-cli/pull/545
https://github.com/particle-iot/particle-cli/pull/544


## How to Test

Use this branch to administer your Particle product devices and report any unexpected or missing behavior. For details on testing specific changes, see the PRs linked above.

:pray::+1:


## Completeness
